extra/exports

```diff
@@ -152,6 +152,7 @@ decimal_from_string
 decimal_unpack
 error_ref
 error_set_prev
+error_unpack_unsafe
```

Comment:
Hm, it seems msgpackffi will simply crash or decode garbage, if will receive a truncated error object. Is there a ticket to introduce a proper `error_unpack()`? The current one is useless, it does not check anything.

Reply:
Yes: https://github.com/tarantool/tarantool/issues/5014